data/lib/src/datasource/shared_space_document_datasource.dart

```diff
@@ -43,4 +43,10 @@ abstract class SharedSpaceDocumentDataSource {
   Future<List<WorkGroupNode>> getAllChildNodes(
       SharedSpaceId sharedSpaceId,
       {WorkGroupNodeId parentNodeId});
+
+  Future<List<WorkGroupNode>> copyToSharedSpace(
+    CopyRequest copyRequest,
+    SharedSpaceId destinationSharedSpaceId,
+    [WorkGroupNodeId destinationParentNodeId]
```

Comment:
Why don't you use `{}` instead of `[]` (`Optional positional argument` ). IMO it's much easier to read.